src/model/y1/y1_8_m12_treat.R

```diff
@@ -25,7 +25,9 @@ y1_m12_modalities_df <- m12_treatment_modalities(input_df = y1_m12_cancer_df,
                                                  chemo_rate = m12_treatment_chemo_y1,
                                                  nadj_imm_rate = m12_treatment_nadj_imm_y1)
 
-y1_m24_FU_input_df <- y1_m12_treat_groups_list[[4]]
+y1_m12_m24_FU_input_df <- y1_m12_treat_groups_list[[4]] 
+
+y1_m24_FU_input_df <- bind_rows(y1_m3_m24_FU_input_df, y1_m24_FU_input_df)
```

Comment:
The `bind_rows` call needs amending as the `y1_m24_FU_input_df` isn't defined. This will need amending to `y1_m12_m24_FU_input_df`